table/clients.go

```diff
@@ -6,12 +6,12 @@ import (
 )
 
 //go:generate moq -out testdata/zebedeeclient.go -pkg testdata . ZebedeeClient
-//go:generate moq -out testdata/tablerendererclient.go -pkg testdata . TableRendererClient
+//go:generate moq -out testdata/tablerendererclient.go -pkg testdata . RendererClient
 
```

Comment:
Why this name change?

Reply:
Again, the linter complained - this time that having TableRendererClient might lead to table.TableRendererClient (which it didn't like) and recommended renaming to RendererClient so it be table.RendererClient.